app/helpers/script_helper.rb

```diff
@@ -0,0 +1,17 @@
+require 'set'
+
+# rubocop:disable Rails/HelperInstanceVariable
+module ScriptHelper
+  def javascript_pack_tag_once(name)
+    @scripts ||= Set.new
+    @scripts.add(name)
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def print_javascript_pack_once_tags
+    return unless @scripts
+    @scripts.map { |name| javascript_pack_tag(name) }.join('').html_safe
```

Comment:
I *think* we can use https://apidock.com/rails/ActionView/Helpers/OutputSafetyHelper/safe_join, and maybe fix the OutputSafety cop?

```suggestion
    safe_join(@scripts.map { |name| javascript_pack_tag(name) }, '')
```

Reply:
Awesome, even better!